apisix/core/config_yaml.lua

```diff
@@ -45,11 +45,16 @@ local pcall        = pcall
 local io           = io
 local ngx          = ngx
 local re_find      = ngx.re.find
+local process      = require("ngx.process")
+local worker_id   = ngx.worker.id
 local apisix_yaml_path = profile:yaml_path("apisix")
 local created_obj  = {}
 local shared_dict
-
-
+local status_report_shared_dict_name = "status-report"
+local is_http = ngx.config.subsystem == "http"
+if not is_http then
+    status_report_shared_dict_name = status_report_shared_dict_name .. "-stream"
+end
```

Comment:
We don't need this separate branch anymore do we?

Reply:
we need separate shared dict for stream